lib/vanagon/engine/always_be_scheduling.rb

```diff
@@ -37,7 +37,21 @@ def name
 
       # return the platform name as the "host" name
       def build_host_name
-        @platform.name
+        platform_name_to_vmpooler_style(@platform.name)
+      end
+
+      def platform_name_to_vmpooler_style(platform)
+        translation = {
+          "aix-5.3-ppc" => "aix-53-ppc",
```

Comment:
We have all sorts of mappings like this in beaker-hostgenerator too.  At some point there's gotta be a shared library these tools can draw from, right?  Not sure.